oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/type/event/Source.java

```diff
@@ -29,4 +30,18 @@
     private String service;
     private String serviceInstance;
     private String endpoint;
+
+    public String getSourcesStr() {
```

Comment:
I am super confused about this logic, every entity has a very clear ID rule, managed by `IDManager` in SkyWalking. Why do you create so many of this kind of method in here and `AlarmMessage`?
The codes seem to be changed randomly.